_data/simple-icons.json

```
@@ -12370,7 +12370,7 @@
         {
             "title": "Ubiquiti",
             "hex": "0559C9",
-            "source": "https://www.ui.com/marketing/#logos"
+            "source": "https://commons.wikimedia.org/wiki/File:Ubiquiti_Logo_2023.svg"
```

There is an [SVG](https://www.ui.com/microsite/static/media/logo.47175193.svg) available (albeit hidden as a `-webkit-mask` in the source code) on their homepage. IMHO that'd be a better source link than Wikimedia.

```suggestion
            "source": "https://www.ui.com"
```